File: src/targets/php/http2/fixtures/headers.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('https://httpbin.org/headers');
$request->setRequestMethod('GET');
$request->setHeaders([
  'accept' => 'application/json',
  'x-foo' => 'Bar',
<<<<<<< HEAD
  'x-bar' => 'Foo'
=======
  'quoted-value' => '"quoted" \'string\''
>>>>>>> upstream/master
]);

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: src/targets/rust/reqwest/fixtures/headers.rs
use reqwest;

#[tokio::main]
pub async fn main() {
    let url = "https://httpbin.org/headers";

    let client = reqwest::Client::new();
    let response = client.get(url)
        .header("accept", "application/json")
        .header("x-foo", "Bar")
        .header("x-bar", "Foo")
        .header("quoted-value", "\"quoted\" 'string'")
        .send()
        .await;

    let results = response.unwrap()
        .json::<serde_json::Value>()
        .await
        .unwrap();

    println!("{}", results);
}
